### PR TITLE
JWK kid generator plus basic api endpoint

### PR DESCRIPTION
**Pull request by kalabiyau**

### CODE REVIEW

File: spec/vendor/kid_generator.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	"github.com/docker/libtrust"
+	"os"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "Usage: kid_generator <public_key>\n")
+		os.Exit(1)
+	}
+	fmt.Println(os.Args)
```
Comment (flavio) [inline spec/vendor/kid_generator.go]: This can be removed, I forgot it...

**Comment by flavio** [comment]
Also let's remove `spec/vendor/kid_viewer`, we will build that on travis CI later.

File: spec/vendor/kid_generator.go (hunk 1 above)
Comment (kalabiyau) [inline spec/vendor/kid_generator.go]: That's I've added to look what is passed, but of course we can remove it

File: app/controllers/api/v2/tokens_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Api::V2::TokensController < Api::BaseController
+
+  def show
+    token_request_params
+    token_request_params.permitted?
```
Comment (flavio) [inline app/controllers/api/v2/tokens_controller.rb]: We totally ignore the return value of `permitted?`. Seems useless.

**Pull request merged**